TwitchLib.Api.Helix/Subscriptions.cs

```diff
@@ -37,7 +37,8 @@ public Task<GetUserSubscriptionsResponse> GetUserSubscriptionsAsync(string broad
             return TwitchGetGenericAsync<GetUserSubscriptionsResponse>("/subscriptions", ApiVersion.Helix, getParams, accessToken);
         }
 
-        public Task<GetBroadcasterSubscriptionsResponse> GetBroadcasterSubscriptions(string broadcasterId, string accessToken = null)
+        public Task<GetBroadcasterSubscriptionsResponse> GetBroadcasterSubscriptions(string broadcasterId, string cursor = null,
```

Comment:
to keep things consistent I would like to have quantity changed to first as variable name